components/clarinet-cli/src/lsp/native_bridge.rs

```diff
@@ -58,7 +58,7 @@ pub async fn start_language_server(
                         LspRequest::Initialize(_) => {
                             process_mutating_request(request, &mut editor_state)
                         }
-                        _ => process_request(request, &editor_state),
+                        _ => process_request(request, &editor_state).await,
```

Comment:
ooouh interesting. does it fully fixes the native bridge?

Reply:
lol. this wasn't supposed to be commited 😓